.circleci/config.yml

```diff
@@ -28,6 +28,29 @@ workflows:
       - test-installation-deb:
           requires:
             - build-deb
+  binary:
+    jobs:
+      - build-binary
+      - test-installation-binary:
+          requires:
+            - build-binary
+      - upload-binary:
+          name: upload-binary-staging
+          context: aws-staging
+          requires:
+            - test-installation-binary
+      - upload-binary:
+          name: upload-binary-production
+          context: aws
+          requires:
+            - upload-binary-staging
+          filters:
+            # Ignore any commit on any branch by default
+            branches:
+              ignore: /.*/
+            # Run only when a tag is created
+            tags:
+              only: /^v.+\..+\..+/
```

Comment:
I dunno, I don't think this is the full semantic version regex :smirk:

Reply:
https://regex101.com/r/Ly7O1x/3/ :eyes:

Reply:
😬